### ISSUE TITLE
How to copy images to xcassets?

### ISSUE THREAD

**Issue by alaughlin** [Malaa-tech/expo-custom-assets#3]
I am attempting to use this library to show native context menus:
https://www.npmjs.com/package/react-native-context-menu-view

This requires custom icons to be added to xcassets. I thought this library would do that, however it just seems to be copying my assets to `ios/MyApp/Assets`?

Is that supposed to be happening with this library? If not, does anyone know how to do this with an expo managed workflow?

**Comment by mzaien** [comment]
Hello @alaughlin thanks for reaching out, currently yes our library does not add assets as an xcassets asset, tho we can implement this as a extra feature but what exaclty do you want it to do just adds assets under Assets.xcassets or what ?

**Comment by lxp-git** [comment]
expo sdk 51 add this feature.

**Comment by mzaien** [comment]
> expo sdk 51 add this feature.

Great can you like the docs for this?

**Comment by christophemenager** [comment]
> expo sdk 51 add this feature.
> 
> [load-an-asset-at-build-time](https://docs.expo.dev/develop/user-interface/assets/#load-an-asset-at-build-time)

@lxp-git  Not true, for now it only supports limited file extensions: https://docs.expo.dev/versions/latest/sdk/asset/, and .`riv` is not one of them

**Comment by mzaien** [comment]
> > expo sdk 51 add this feature.
> > [load-an-asset-at-build-time](https://docs.expo.dev/develop/user-interface/assets/#load-an-asset-at-build-time)
> 
> @lxp-git Not true, for now it only supports limited file extensions: https://docs.expo.dev/versions/latest/sdk/asset/, and .`riv` is not one of them

Thanks for adding the reference
